src/stories/temp-converter.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState, useEffect } from "react";
+
+export type TempConverterProps = {
+    items: string[];
+};
+
+export default function TempConverter(props: TempConverterProps) {
+    const [startUnit, setStartUnit] = useState<string>("");
+    const [endUnit, setEndUnit] = useState<string>("");
+    const [temperature, setTemperature] = useState<string>("");
```

Comment:
`temperature` is not a string. It's a floating point number. Same with `convertedTemp`.

*Tip:*  
`NaN` is also of type `number` (although the name says differently :) ).